### PR TITLE
Update usage info

### PR DESCRIPTION
**Pull request by mahesh-panchal**
Updates suggestion for writing custom configuration.

## PR checklist

 - [ ] This comment contains a description of changes (with reason)
 - [ ] `CHANGELOG.md` is updated
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by mashehu — APPROVED** [review]
Is it possible that you don't have the latest version of `yamllint.yml`? Because the template linting errors were fixed in the latest version.

**Comment by mahesh-panchal** [comment]
I don't know what's going on with the yaml linting there. I merged dev on my fork, opened up a Gitpod, made the change, and made the pull request.

**Comment by mashehu** [comment]
yeah, yamllint inside the gitpod is fine... weird.

File: nf_core/pipeline-template/docs/usage.md [hunk 1]
```diff
@@ -179,14 +179,14 @@ The custom config below can then be provided to the pipeline via the [`-c`](#-c)
 
 ```nextflow
 process {
-    withName: STAR_ALIGN {
+    withName: 'RNASEQ:ALIGN_STAR:STAR_ALIGN' {
```
Comment (drpatelh) [inline nf_core/pipeline-template/docs/usage.md:182]:
```suggestion
    withName: 'NFCORE_RNASEQ:RNASEQ:ALIGN_STAR:STAR_ALIGN' {
```

**Pull request merged**